python/robyn/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: robyn.tech
+  github: sparckles/robyn
+  version: 0.54.0
+
+  engines:
+    - robyn
+
+language:
+  engines:
+    robyn:
+      command: python /usr/src/app/server.py
```

Comment:
Hey @jackcvr 👋 

The author of Robyn here. I am not familiar with benchmark config, but you may also want to specify the workers and processes to get the maximum throughput.

`python /usr/src/app/server.py --workers=N --processes=M`

Reply:
Hey @sansyrox 
Could you suggest optimal amount of N and M?

Reply:
The more accurate config is a value that we can set for a production ready app. We try not to tweak benchmark for performances but display the more realistic performance like in production

Reply:
Hey @jackcvr @jackcvr , yes sure :D

So, the exact numbers requires some fine-tuning in a real world scenario but a good estimate to start is

N = `os.cpu_count()`
M = `number of cores`

and there is an additional flag that you would run in production mode is `--log-level=WARN` - this is for internal robyn logs

So 
`python /usr/src/app/server.py --workers=N --processes=M --log-level=WARN`